NitroxClient/Communication/Packets/Processors/VehicleDockingProcessor.cs

```diff
@@ -28,10 +35,15 @@ public override void Process(VehicleDocking packet)
 
             Vehicle vehicle = vehicleGo.RequireComponent<Vehicle>();
             VehicleDockingBay vehicleDockingBay = vehicleDockingBayGo.RequireComponentInChildren<VehicleDockingBay>();
-            
+
             using (packetSender.Suppress<VehicleDocking>())
             {
-                vehicleDockingBay.SetVehicleDocked(vehicle);
+                Log.Debug($"Set vehicle docked for {vehicleDockingBay.gameObject.name}");
+                vehicle.GetComponent<MultiplayerVehicleControl>().Exit();
+                Log.Debug($"before dock isKinematic: {vehicle.useRigidbody.isKinematic}");
+                vehicleDockingBay.DockVehicle(vehicle);
+                vehicle.useRigidbody.isKinematic = false;
```

Comment:
Let's add a comment saying that the ordering is important because the function above changed isKinematic.